applications/nvidia_video_codec/nvc_encode_decode/python/README.md

```diff
@@ -0,0 +1,219 @@
+# NVIDIA Video Codec
+
+This application demonstrates the use of NVIDIA Video Codec SDK. The application loads a video file, encodes the video using either H.264 or HEVC (H.265), decodes the video, and displays it with Holoviz.
+
+### Requirements
+
+- x86 and SBSA platforms with dedicated GPU
+- [NVIDIA Video Codec SDK](https://developer.nvidia.com/video-codec-sdk)
+
+> 💡 Note: NVIDIA IGX Orin with integrated GPU is not currently supported.
+
+### Data
+
+[📦️ (NGC) Sample App Data for AI-based Endoscopy Tool Tracking](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara-holoscan/resources/holoscan_endoscopy_sample_data)
+
+
+## Building and Running the NVIDIA Video Codec Application
+
+### Python
+
+```bash
+./holohub run nvc_encode_decode --language python --docker-file applications/nvidia_video_codec/Dockerfile
```

Comment:
`--docker-file` value here should already be the default specified in `metadata.json`, do you see otherwise?

```suggestion
./holohub run nvc_encode_decode --language python
```